spec/mongoid/reloadable_spec.rb

```diff
@@ -353,5 +353,82 @@
         expect(reloaded.readonly?).to be(false)
       end
     end
+
+    context 'on sharded cluster' do
+      require_topology :sharded
+
+      before(:all) do
+        CONFIG[:clients][:other] = CONFIG[:clients][:default].dup
+        CONFIG[:clients][:other][:database] = 'other'
+        Mongoid::Clients.clients.values.each(&:close)
+        Mongoid::Config.send(:clients=, CONFIG[:clients])
+        Mongoid::Clients.with_name(:other).subscribe(Mongo::Monitoring::COMMAND, EventSubscriber.new)
+      end
+
+      let(:subscriber) do
+        client = Mongoid::Clients.with_name(:other)
+        monitoring = if client.respond_to?(:monitoring, true)
+          client.send(:monitoring)
+        else
+          # driver 2.5
```

Comment:
According to gemspec, Mongoid presently requires driver 2.7+, thus this branch should be removable (everywhere).